### ISSUE TITLE
Links are broken ("undefined" in URL)

### ISSUE THREAD

**Issue by medicinal-matt** [MishaKav/pytest-coverage-comment#39]
Hi!

We really like this action. We just have one small problem. 

I'm not sure if I have misconfigured something, but it seems to me that links are broken in our setup. Could it be because we are on a private Github repository?

Instead of 

`https://github.com/<COMPANY>/<REPO>/blob/<COMMIT>/file.py`

they go

`https://github.com/undefined/blob/<COMMIT>/file.py`

Is there a way we can fix this?

**Comment by MishaKav** [comment]
Can you please post more details on that?
maybe configuration of workflow.yaml, maybe the full error? maybe you use some event (like push, pull_request, that action didn't support)
I and many other people run this action on private repos, and they are fine.

**Comment by medicinal-matt** [comment]
Ohh, I assumed that would be it because it is sometimes hidden when you are not logged in and it sounded related.

I'll rephrase the issue then. There is no error. All the links in the comment that posts are simply broken.

This is how it is configured in the workflow.yaml
```yaml
name: CI/CD Pipeline
on:
  push:
    branches:
    - main
  pull_request:

jobs:
  ci:
    strategy:
      matrix:
        type: [
          'lint_python',
          'lint_sql',
          'test_file_tasks',
          'test_python',
          'validate_terraform'
        ]

    runs-on: ubuntu-latest

    steps:
    
[...]

    - name: Run Pytest
      if: matrix.type == 'test_python'
      run: |
        docker-compose run etl \
          python -m pytest \
          --cov=src --cov-fail-under=40 --cov-report=term-missing:skip-covered --junitxml=pytest.xml > pytest-coverage.txt

    - name: Post Pytest Coverage Comment
      if: matrix.type == 'test_python'
      uses: MishaKav/pytest-coverage-comment@main
      with:
        pytest-coverage-path: ./pytest-coverage.txt
        junitxml-path: ./pytest.xml
```

**Comment by medicinal-matt** [comment]
I also came to learn that you have to specify `--cov-report=term-missing`? Otherwise the table seems to be broken as it is still expecting the `Missing` column

**Comment by medicinal-matt** [comment]
It obviously seems like the repository would come from https://github.com/MishaKav/pytest-coverage-comment/blob/v1.1.14/src/parse.js#L291, what would make `${options.repository}` be `undefined`? I don't know JavaScript.

**Comment by MishaKav** [comment]
> It obviously seems like the repository would come from https://github.com/MishaKav/pytest-coverage-comment/blob/v1.1.14/src/parse.js#L291, what would make ${options.repository} be undefined? I don't know JavaScript.

Can you please try to rerun it, there was a contribution to fix that #40.
just run `MishaKav/pytest-coverage-comment@main` or use `MishaKav/pytest-coverage-comment@v1.1.15`

**Comment by MishaKav** [comment]
>I also came to learn that you have to specify --cov-report=term-missing? Otherwise the table seems to be broken as it is still expecting the Missing column

You are right, it's a bug. Fixed it #42 and release `v1.1.16`
